### PR TITLE
unit test refactoring for master & unit test for identities keeper

### PR DESCRIPTION
**Pull request by 0xankit**
## Description

Closes: #XXXX

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and please add links to any
relevant follow-up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json)
  in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (
  see [PR Targeting](https://github.com/AssetMantle/modules/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines
  for [building modules](https://github.com/AssetMantle/modules/blob/master/docs/building-modules)
- [ ] included the necessary unit and
  integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add your handle next to the items
reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the
  correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: modules/identities/internal/transactions/define/keeper_test.go [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+	"github.com/AssetMantle/modules/modules/maintainers/auxiliaries/super"
+	"github.com/AssetMantle/modules/modules/metas/auxiliaries/supplement"
+	"github.com/AssetMantle/modules/schema"
+	baseData "github.com/AssetMantle/modules/schema/data/base"
+	"github.com/AssetMantle/modules/schema/documents/base"
+	"github.com/AssetMantle/modules/schema/errors/constants"
+	"github.com/AssetMantle/modules/schema/helpers"
+	baseHelpers "github.com/AssetMantle/modules/schema/helpers/base"
+	base3 "github.com/AssetMantle/modules/schema/ids/base"
+	base2 "github.com/AssetMantle/modules/schema/lists/base"
```
Comment (muku314115) [inline modules/identities/internal/transactions/define/keeper_test.go:22]: Can we change this to baseLists and apply such changes for other imports as well

File: modules/identities/internal/transactions/issue/keeper_test.go [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+	"github.com/AssetMantle/modules/modules/identities/internal/mappable"
+	"github.com/AssetMantle/modules/modules/identities/internal/parameters"
+	"github.com/AssetMantle/modules/modules/maintainers/auxiliaries/verify"
+	"github.com/AssetMantle/modules/schema"
+	baseData "github.com/AssetMantle/modules/schema/data/base"
+	"github.com/AssetMantle/modules/schema/documents/base"
+	errorConstants "github.com/AssetMantle/modules/schema/errors/constants"
+	"github.com/AssetMantle/modules/schema/helpers"
+	baseHelpers "github.com/AssetMantle/modules/schema/helpers/base"
+	base3 "github.com/AssetMantle/modules/schema/ids/base"
```
Comment (muku314115) [inline modules/identities/internal/transactions/issue/keeper_test.go:20]: rename please

**Review by muku314115 — CHANGES_REQUESTED** [review]
imports need to follow proper nomenclature

**Review by deepanshutr — APPROVED** [review]
Reviewed.

**Review by muku314115 — APPROVED** [review]

**Pull request merged**